src/test/ui/lifetime-errors/ex3-both-anon-regions-latebound-regions.stderr

```diff
@@ -0,0 +1,10 @@
+error[E0623]: lifetime mismatch
+  --> $DIR/ex3-both-anon-regions-latebound-regions.rs:12:12
+   |
+11 | fn foo<'a,'b>(x: &mut Vec<&'a u8>, y: &'b u8) {
+   |                           ------      ------ these two types are declared with different lifetimes...
+12 |     x.push(y);
+   |            ^ ...but data from `y` flows into `x` here
+
+error: aborting due to previous error
+
```

Comment:
Also add a test for early-bound regions?

Reply:
The example we had discussed with where clause and sized trait doesn't seem to be working. So.still trying to get the code working for early bound